pkg/virt-controller/services/template.go

```diff
@@ -1135,7 +1135,6 @@ func getRequiredCapabilities(vmi *v1.VirtualMachineInstance) []k8sv1.Capability
 	if (len(vmi.Spec.Domain.Devices.Interfaces) > 0) ||
 		(vmi.Spec.Domain.Devices.AutoattachPodInterface == nil) ||
 		(*vmi.Spec.Domain.Devices.AutoattachPodInterface == true) {
-		res = append(res, CAP_NET_ADMIN)
```

Comment:
IIUC, we need to remove `CAP_NET_RAW` as well.
I think that unprivileged pods have `CAP_NET_RAW`  so it is not enough to remove the cap from the here. We need to add it to the list of dropped capabilities.

Reply:
You are a lot braver than I am; I'd rather tackle one problem at a time.